### ISSUE TITLE
pip install not working

### ISSUE THREAD

**Issue by sn1020** [InstantDomain/instant-segment#32]
Hi,

`pip install instant-segment` doesn't seem to be working for me with Python 3.10.0 (and pip 22.2.2) on a Mac. I'm getting the following error:
```ERROR: Could not find a version that satisfies the requirement instant-segment (from versions: none)
ERROR: No matching distribution found for instant-segment```

**Comment by djc** [comment]
Do you have an Intel or Apple Silicon Mac?

**Comment by sn1020** [comment]
Apple Silicon

**Comment by djc** [comment]
I've published a new version 0.1.5, with wheels for Python 3.10 for all of Linux, macOS on x86-64 and macOS on Apple Silcon.
